src/components/book.js

```diff
@@ -11,14 +14,38 @@ const Book = ({ singleBook }) => {
     await dispatch(getApiBooks());
   };
   return (
-    <section className="list-wrap">
-      <div className="book-info">
-        <div className="title">{singleBook.title}</div>
-        <div className="author">{singleBook.author}</div>
-        <div className="category">{singleBook.category}</div>
-      </div>
-      <Button className="remove-btn" onClick={() => handleRemove()} label="Remove" />
-    </section>
+    <li className="list-wrap">
+      <article className="book-details">
+        <div className="book-info">
+          <p className="category">{singleBook.category}</p>
+          <p className="title">{singleBook.title}</p>
+          <p className="author">{singleBook.author}</p>
+        </div>
+        <div className="all-btn">
+          <Button className="comment-btn" label="Comments" />
+          <div className="vertical-divider" />
+          <Button className="remove-btn" onClick={() => handleRemove()} label="Remove" />
+          <div className="vertical-divider" />
+          <Button className="edit-btn" label="Edit" />
+        </div>
+      </article>
+      <article className="progressbar-wrap">
+        <div className="circle-bar">
+          <CircularProgressbar value={45} />
+        </div>
```

Comment:
- [ ] Please change the color of your circular progress bar the color is not the same as the Zaplin design so kindly change the color of your circular progress bar.

**Your Design**
![Screenshot from 2023-07-20 19-51-50](https://github.com/zuhebahmed88091/Bookstore/assets/45226848/a8630190-c770-45aa-b070-c8189b16dda2)

**Zaplin Design**
![Screenshot from 2023-07-20 19-10-41](https://github.com/zuhebahmed88091/Bookstore/assets/45226848/8167cb06-7482-4cbf-8751-d666d1bf9158)

Reply:
I have copied the bar color->#307bbe from zeplin.
Issue resolved